cmake/abseil-cpp.cmake

```diff
@@ -4,6 +4,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+unset(_abseil_cxx_standard)
+
+if(DEFINED CURRENT_CXX_STANDARD AND NOT CURRENT_CXX_STANDARD STREQUAL "")
+  set(_abseil_cxx_standard
+      -DABSL_CXX_STANDARD=${CURRENT_CXX_STANDARD}
+      -DCMAKE_CXX_STANDARD=${CURRENT_CXX_STANDARD}
```

Comment:
So even though CMAKE_CXX_STANDARD is modified as part of abseil.cmake, it's global variable and applicable to other cmakes as well?

Reply:
Suresh's CXX_STANDARD option (a workaround for build issues in Fedora 37) only applies to the Abseil and gRPC subbuilds. It's not global.